src/components/Votes/VoteButtons/AddVoteButton.tsx

```diff
@@ -22,7 +25,7 @@ export const AddVoteButton: FC<AddVoteProps> = ({noteId, disabled, disabledReaso
 
   return (
     <DotButton
-      className="vote-button-add"
+      className={classNames("vote-button-add", colorClassName && needsHighContrast(colorClassName) && "vote-button-add--high-contrast")}
```

Comment:
```suggestion
      className={classNames("vote-button-add", {"vote-button-add--high-contrast": colorClassName && needsHighContrast(colorClassName)})}
```


would be more readable and streamlined with the rest. maybe you could even allow undefined in the `needHighContrast` func and remove `colorClassName &&`